### PR TITLE
chore(cce): fix api comments of CCE

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

fix api comments of CCE

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
=== RUN   TestAccSomethingV0_basic
--- PASS: TestAccSomethingV0_basic (70.75s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       70.796s
```

### CODE REVIEW

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go [hunk 1]
```diff
@@ -29,15 +29,25 @@ import (
 
 // ResourceCCEClusterV3 defines the CCE cluster resource schema and functions.
 // Deprecated: It's a deprecated function, please refer to the function 'ResourceCluster'.
-// @API CCE DELETE /api/v3/projects/{project_id}/clusters/{id}
-// @API CCE GET /api/v3/projects/{project_id}/clusters/{id}
-// @API CCE PUT /api/v3/projects/{project_id}/clusters/{id}
+
 // @API CCE POST /api/v3/projects/{project_id}/clusters
```
Comment (ShiChangkuo) [inline huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go:30]: it seems that we should add those API comments before #L30

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go [hunk 2]
```diff
@@ -29,15 +29,25 @@ import (
...
 // @API CCE PUT /api/v3/projects/{project_id}/clusters/{id}/mastereip
-// @API CCE POST /api/v3/projects/{project_id}/clusters/{id}/operation/{action}
 // @API CCE POST /api/v3/projects/{project_id}/clusters/{id}/tags/{action}
-// @API AOM POST /svcstg/icmgr/v1/{project_id}/agents
+// @API BSS POST /v2/orders/subscriptions/resources/autorenew/{id}
+// @API BSS DELETE /v2/orders/subscriptions/resources/autorenew/{id}
+// @API EPS POST /v1.0/enterprise-projects/{enterprise_project_id}/resources-migrate
+// @API EPS POST /v1.0/enterprise-projects/{enterprise_project_id}/resources/filter
+// @API CCE DELETE /api/v3/projects/{project_id}/clusters/{id}
+// @API BSS POST /v2/orders/subscriptions/resources/unsubscribe
```
Comment (ShiChangkuo) [inline huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: it's very appreciate to sort them by product and CRUD

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go (hunk 1 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go:30]: fixed

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: sure

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go (hunk 1 above)
Comment (Jason-Zhang9309) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go:30]: fixed

**Pull request merged**